dist/api_data.py

```diff
@@ -1162,6 +1162,16 @@ def __cmp__(self, other):
         type='list'),
 ]),
 
+'WT_SESSION.query_timestamp' : Method([
+    Config('get', 'read', r'''
+        specify which timestamp to query: \c read returns the timestamp the
+        transaction is reading at.  \c commit returns the most recently set
```

Comment:
Please rephrase this to be `\c read returns the timestamp at which the transaction is reading.`